File: solutions/2148-minimum-number-of-moves-to-seat-everyone/solution.java
class Solution {
    public int minMovesToSeat(int[] seats, int[] students) {
        Arrays.sort(seats);
        Arrays.sort(students);
        int sum1=0;
        for(int i=0;i<seats.length;i++)
            sum1+=Math.abs(students[i]-seats[i]);
        return sum1;
    }
}

File: solutions/2148-minimum-number-of-moves-to-seat-everyone/solution.py
class Solution(object):
    def minMovesToSeat(self, seats, students):
        """
        :type seats: List[int]
        :type students: List[int]
        :rtype: int
        """
        seats=sorted(seats)
        students=sorted(students)
        sum1=0
        for i in range(len(seats)):
            sum1+=abs(seats[i]-students[i])
        return sum1

